e2e/suites.go

```diff
@@ -177,7 +175,7 @@ func (s *BackendTestSuite) EnsureNodeSync(forceResync ...bool) {
 	require := s.Require()
 
 	ethClient := s.LightEthereumService()
-	sync := node.NewSyncPoll(ethClient)
+	sync := node.NewSyncPoll(ethClient.Downloader())
```

Comment:
Why direct use of `node.NewSyncPoll` here? You could parse `s.EnsureNodeSync(true)` to force sync validation again.